date-mod.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+var myObj = (function iife(){
+  //sets the day of the month to the date object
+  function setDate(){
+    var sd = new Date();
+    
+    return sd.setDate(15);
+  }
+  //method returns the day of the week (from 0 to 6) for the specified date.
+  function getDayName(){
+    var d = new Date();
+    var weekday = new Array(7);
```

Comment:
While this works, it's preferable to use array literals rather than instantiate new array objects like this:

`var weekday = ['Sunday', 'Monday', 'Tuesday',...];`